frontend/src/Episode/Summary/EpisodeAiring.js

```diff
@@ -31,7 +31,7 @@ function EpisodeAiring(props) {
   if (!airDateUtc) {
     return (
       <span>
-        {translate('AirsTbaOn', { networkLabel: '' })}networkLabel
+        {translate('AirsTbaOn', { networkLabel: '' })}{networkLabel}
```

Comment:
Shouldn't this be?

```suggestion
        {translate('AirsTbaOn', { networkLabel })}
```

Reply:
No, it should be the same as the rest of them, I just missed the curly braces om this case. TBA episodes can still have a NetworkLabel. We did it this was for now as translate can't handle objects.